community/sample-apps/nextjs-8-serverless/README.md

```diff
@@ -0,0 +1,125 @@
+
+  Verify if the row is inserted successfully
+
+  ![Insert data into author table](../gatsby-postgres-graphql/assets/browse_rows.jpg)
+
+- Clone this repo:
+  ```bash
+  git clone https://github.com/hasura/graphql-engine
+  cd graphql-engine/community/sample-apps/nextjs-8-serverless
+  ```
+
+- Install npm modules:
+  ```bash
+  npm install
+  ```
+
+- Open `lib/init-apollo.js` and configure Hasura's GraphQL Endpoint as follows:
+
+  ```js
+
+    function create (initialState) {
+      return new ApolloClient({
+        connectToDevTools: process.browser,
+        ssrMode: !process.browser, // Disables forceFetch on the server (so queries are only run once)
+        link: new HttpLink({
+          uri: 'https://myapp.herokuapp.com/v1alpha1/graphql', // Server URL (must be absolute)
+          credentials: 'same-origin' // Additional fetch() options like `credentials` or `headers`
+        }),
+        cache: new InMemoryCache().restore(initialState || {})
+      })
+    }
+
+  ```
+Replace the `uri` with your own Hasura GraphQL endpoint.
+
+In this example, we integrate Apollo with Next by wrapping our *pages/_app.js* inside a higher-order component HOC. Using the HOC pattern we're able to pass down a central store of query result data created by Apollo into our React component hierarchy defined inside each page of our Next application.
+
+On initial page load, while on the server and inside `getInitialProps`, we invoke the Apollo method,  [`getDataFromTree`](https://www.apollographql.com/docs/react/features/server-side-rendering.html#getDataFromTree). This method returns a promise; at the point in which the promise resolves, our Apollo Client store is completely initialized.
+
+- We have defined the graphql query in `components/AuthorList.js`. 
+
+    ```graphql
+
+    query author($skip: Int!) {
+        author(offset: $skip, limit: 5) {
+          id
+          name
+        }
+        author_aggregate {
+          aggregate {
+            count
+          }
+        }  
+    }
+
+    ```
+
+- Run the app:
+  ```bash
+  npm run dev
+  ```
+- Test the app
+  Visit [http://localhost:3000](http://localhost:3000) to view the app
+
+# Serverless Mode
+
+With Next.js 8, each page in the `pages` directory becomes a serverless lambda. To enable `serverless` mode, we add the `serverless` build `target` in `next.config.js`.
+
+```
+module.exports = {
+  target: "serverless",
+};
+```
+
+That's it! Now build the serverless app by running the following command:
+
+```
+$ npm run build
+```
+
+In the `.next` folder, you will see a `serverless` folder generated after the build. Inside that there is a `pages` folder, which will have outputs of lambda per page.
+
+```
+pages/index.js => .next/serverless/pages/index.js
+pages/about.js => .next/serverless/pages/about.js
+```
+
+# Deploy to now.sh
+
+Deploy it to the cloud with [now](https://zeit.co/now) ([download](https://zeit.co/download)):
+
+```bash
+$ npm install -g now
+$ now
```

Comment:
```suggestion
now
```

Reply:
The idea was to distinguish it as command line vs a code snippet.